nng/src/options/types.rs

```diff
@@ -17,13 +17,13 @@ create_option! {
     ///     * TLS
     /// * Pipes can read from this on the following transports:
     ///     * IPC
-    ///     * InProc
+    ///     * `InProc`
     ///     * TCP
-    ///     * ZeroTier
+    ///     * `ZeroTier`
```

Comment:
Not something you have to fix, just another FYI: All of the ZeroTier support in NNG is considered deprecated.We should update the documents and the bindings to reflect that. See [man 3 nng_zt_register](https://nng.nanomsg.org/man/v1.10.0/nng_zt_register.3.html).